src/bosh-director/lib/bosh/director/api/config_manager.rb

```diff
@@ -13,6 +13,15 @@ def create(type, name, config_yaml, team_id = nil)
             config.save
           end
 
+          def deploy_config_enabled?
+            deploy_config = find(type: 'deploy', limit: 999)
+
+            deploy_config.each do |config|
+              return true
+            end
+            false
```

Comment:
May it is better to write it something like this:
return !deploy_config.empty?

Reply:
Thanks! Leftover from an old approach.